<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateWidget</class>
 <widget class="QWidget" name="CreateWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>973</width>
    <height>652</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="OrderSelectLayout" stretch="3,1,0">
   <item>
    <widget class="QWidget" name="MapWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>350</height>
      </size>
     </property>
     <property name="font">
      <font>
       <family>Segoe UI</family>
      </font>
     </property>
     <layout class="QVBoxLayout" name="MapLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableView" name="selectedTable">
     <property name="styleSheet">
      <string notr="true">background-color: rgb(200, 200, 200);</string>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="selectionBehavior">
      <enum>QAbstractItemView::SelectRows</enum>
     </property>
     <property name="verticalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="horizontalScrollMode">
      <enum>QAbstractItemView::ScrollPerPixel</enum>
     </property>
     <property name="sortingEnabled">
      <bool>true</bool>
     </property>
     <property name="cornerButtonEnabled">
      <bool>false</bool>
     </property>
     <attribute name="horizontalHeaderMinimumSectionSize">
      <number>140</number>
     </attribute>
     <attribute name="verticalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
     <attribute name="verticalHeaderMinimumSectionSize">
      <number>30</number>
     </attribute>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="SortWidget" native="true">
     <layout class="QHBoxLayout" name="bottomPanel2Layout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignVCenter">
       <widget class="QLabel" name="scheduleDateLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="text">
         <string>Schedule Date:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDateEdit" name="dateEdit">
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="toolTip">
         <string>Schedule execution date</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="minimumDate">
         <date>
          <year>2022</year>
          <month>1</month>
          <day>1</day>
         </date>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hspacer5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>6</width>
          <height>37</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QLabel" name="driverLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="text">
         <string>Driver:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="driverSearch">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="toolTip">
         <string>Search by driver</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>Doe</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Driver2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Driver3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="hspacer6">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>6</width>
          <height>37</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QLabel" name="vehicleLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="text">
         <string>Vehicle:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="vehicleSearch">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="toolTip">
         <string>Search by vehicle</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <item>
         <property name="text">
          <string>Truck</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vehicle2</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Vehicle3</string>
         </property>
        </item>
       </widget>
      </item>
      <item>
       <spacer name="hspacer7">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>6</width>
          <height>37</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QLabel" name="driveHrsLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="text">
         <string>Drive Hrs:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="driverHrsCount">
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="toolTip">
         <string>Estimated drive time</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hspacer8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>6</width>
          <height>37</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QLabel" name="itemsLabel">
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="text">
         <string>Items:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinBox">
        <property name="font">
         <font>
          <family>Segoe UI</family>
         </font>
        </property>
        <property name="toolTip">
         <string>Total nursery items</string>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgb(255, 255, 255);</string>
        </property>
        <property name="buttonSymbols">
         <enum>QAbstractSpinBox::NoButtons</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hspacer9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Fixed</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>6</width>
          <height>37</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCommandLinkButton" name="optimizeButton">
        <property name="maximumSize">
         <size>
          <width>93</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <pointsize>8</pointsize>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Optimize Route</string>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Optimize</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>D:/jacbo/Downloads/speedometer2.png</normaloff>D:/jacbo/Downloads/speedometer2.png</iconset>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="hspacer10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
       </spacer>
      </item>
      <item alignment="Qt::AlignVCenter">
       <widget class="QCommandLinkButton" name="submitButton">
        <property name="maximumSize">
         <size>
          <width>100</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Segoe UI</family>
          <weight>50</weight>
          <bold>false</bold>
         </font>
        </property>
        <property name="toolTip">
         <string>Submit schedule</string>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="text">
         <string> Submit</string>
        </property>
        <property name="icon">
         <iconset>
          <normaloff>D:/jacbo/Downloads/submit2.png</normaloff>D:/jacbo/Downloads/submit2.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
